g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/ParameterValidationUtility.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.i18n.api.base.utils;
+
+
+import org.apache.commons.lang3.StringUtils;
+import com.vmware.vip.common.constants.ValidationMsg;
+import com.vmware.vip.common.exceptions.VIPAPIException;
+import com.vmware.vip.common.i18n.dto.TranslationWithPatternDTO;
+import com.vmware.vip.common.utils.RegExpValidatorUtils;
+
+public class ParameterValidationUtility{
+	private ParameterValidationUtility() {}
```

Comment:
remove it, it's not required.

Reply:
Yes, I agree with you. But the Sonarqube scan it as a issue